src/libs/config.js

```diff
@@ -12,4 +12,5 @@ export const getConfig = () => {
  * Flags for hidden features
  */
 export const isCromwellAppVisible = () => getConfig().isCromwellAppVisible
-export const isDataBrowserFrontPage = () => false
+export const isDataTableProvenanceEnabled = () => getConfig().isDataTableProvenanceEnabled
+export const isDataTableVersioningEnabled = () => getConfig().isDataTableVersioningEnabled
```

Comment:
These functions have been removed in the dev branch and should not be re-added here.